standard_schema/prerelease/HED8.3.0.mediawiki

```diff
@@ -208,26 +208,21 @@ Each term in this vocabulary has a human-readable description and may include ad
-**** Lower-extremity <nowiki>{hedId=HED_0012195} [Refers to the whole inferior limb (leg and/or foot).]</nowiki>
-***** Ankle <nowiki>{hedId=HED_0012196} [A gliding joint between the distal ends of the tibia and fibula and the proximal end of the talus.]</nowiki>
-***** Calf <nowiki>{hedId=HED_0012197} [The fleshy part at the back of the leg below the knee.]</nowiki>
-***** Foot <nowiki>{hedId=HED_0012198} [The structure found below the ankle joint required for locomotion.]</nowiki>
-****** Big-toe <nowiki>{hedId=HED_0012199} [The largest toe on the inner side of the foot.]</nowiki>
-****** Heel <nowiki>{hedId=HED_0012200} [The back of the foot below the ankle.]</nowiki>
-****** Instep <nowiki>{hedId=HED_0012201} [The part of the foot between the ball and the heel on the inner side.]</nowiki>
-****** Little-toe <nowiki>{hedId=HED_0012202} [The smallest toe located on the outer side of the foot.]</nowiki>
-****** Toes <nowiki>{hedId=HED_0012203} [The terminal digits of the foot.]</nowiki>
-***** Knee <nowiki>{hedId=HED_0012204} [A joint connecting the lower part of the femur with the upper part of the tibia.]</nowiki>
-***** Shin <nowiki>{hedId=HED_0012205} [Front part of the leg below the knee.]</nowiki>
-***** Thigh <nowiki>{hedId=HED_0012206} [Upper part of the leg between hip and knee.]</nowiki>
+**** Neck <nowiki>{hedId=}[The part of the body connecting the head to the torso, containing the cervical spine and vital pathways of nerves, blood vessels, and the airway.]</nowiki>
 **** Torso <nowiki>{hedId=HED_0012207} [The body excluding the head and neck and limbs.]</nowiki>
-***** Buttocks <nowiki>{hedId=HED_0012208} [The round fleshy parts that form the lower rear area of a human trunk.]</nowiki>
-***** Gentalia <nowiki>{deprecatedFrom=8.1.0, hedId=HED_0012209} [The external organs of reproduction. Deprecated due to spelling error. Use Torso/Genitalia.]</nowiki>
-***** Hip <nowiki>{hedId=HED_0012210} [The lateral prominence of the pelvis from the waist to the thigh.]</nowiki>
-***** Torso-back <nowiki>{hedId=HED_0012211} [The rear surface of the human body from the shoulders to the hips.]</nowiki>
+**** Torso-part <nowiki>{hedId=} [A part of the torso.]</nowiki>
 ***** Torso-chest <nowiki>{hedId=HED_0012212} [The anterior side of the thorax from the neck to the abdomen.]</nowiki>
+***** Torso-back <nowiki>{hedId=HED_0012211} [The rear surface of the human body from the shoulders to the hips.]</nowiki>
 ***** Viscera <nowiki>{hedId=HED_0012213} [Internal organs of the body.]</nowiki>
+***** Abdomen <nowiki>{hedId=}[The part of the body between the thorax and the pelvis.]</nowiki>
+***** Navel <nowiki>{hedId=}[The central mark on the abdomen created by the detachment of the umbilical cord after birth.]</nowiki>
 ***** Waist <nowiki>{hedId=HED_0012214} [The abdominal circumference at the navel.]</nowiki>
+***** Pelvis <nowiki>{hedId=}[The bony structure at the base of the spine supporting the legs.]</nowiki>

```

Comment:
is Pelvis part of Torso --- I thought not.  Maybe Pelvis needs to be one level up. and the things below (Buttocks etc. have one less *)?

Reply:
Overall, yes, the Pelvis is part of the torso ([Wikipedia](https://en.wikipedia.org/wiki/Pelvis#cite_note-FOOTNOTEMoore2014357–8-1)). However, the level depends on the model we want to consider. The literature divides the torso into one, two, three, or more segments. The current suggestion is based on a three-segment torso (chest, abdomen, and pelvis).

I tried to limit the definitions to the terms used in the schema. For example, we have to define Navel, since it is used in the definition of Waist.

Reply:
okay I just didn't know.